demos/tests/cases.py

```diff
@@ -113,6 +113,12 @@ def single_option_cases(key, *args):
         TestCase(options={'-m': ModelArg('human-pose-estimation-0001')}),
     )),
 
+    NativeDemo(subdirectory='imagenet_classification_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'-no_show': None,
+            '-i': ImageDirectoryArg('imagenet-classification')}),
+        TestCase(options={'-m': ModelArg('alexnet')}),
```

Comment:
Please, add some other models. I don't think we need the whole list here.

Reply:
Done

Reply:
You can reuse `single_option_cases` like in https://github.com/opencv/open_model_zoo/blob/b0dc95c5cb5ed00277e21f88dd640f2e4b457dd5/demos/tests/cases.py#L144

Reply:
Done